packages/core/js-client/src/jsServiceHost/interfaces.ts

```diff
@@ -94,7 +94,7 @@ export interface ParticleContext {
     /**
      * Particle's signature
      */
-    signature?: string;
+    signature: Uint8Array;
```

Comment:
Should be back to `string` in base64, right?

Reply:
No, why?